src/components/PeopleFilters/PeopleFilters.tsx

```diff
@@ -0,0 +1,149 @@
+import classNames from 'classnames';
+import { Link, useSearchParams } from 'react-router-dom';
+import { SexFilter } from '../../types/SexFilter';
+import { getSearchWith } from '../../utils/searchHelper';
+import { Params } from '../../types/Params';
+
+const CENTURIES = [16, 17, 18, 19];
+
+export const PeopleFilters = () => {
+  const [searchParams, setSearchParams] = useSearchParams();
+  const query = searchParams.get('query') || '';
+  const centuries = searchParams.getAll('century');
+  const sex = searchParams.get('sex');
+
+  function toggleCentury(data: number) {
+    const century = data.toString();
+
+    const params = new URLSearchParams(searchParams);
+
+    const newCenturies = centuries.includes(century)
+      ? centuries.filter(item => item !== century)
+      : [...centuries, century];
+
+    params.delete('century');
+
+    newCenturies.forEach(el => {
+      params.append('century', el);
+    });
+
+    setSearchParams(params);
+  }
+
+  function setSearchWith(params: Params) {
+    const search = getSearchWith(params, searchParams);
+
+    setSearchParams(search);
+  }
+
+  return (
+    <nav className="panel">
+      <p className="panel-heading">Filters</p>
+
+      <p
+        className="panel-tabs"
+        data-cy="SexFilter"
+      >
+        <Link
+          type="button"
+          className={classNames({ 'is-active': !sex })}
+          to={{
+            search: getSearchWith(
+              { sex: +SexFilter.all || null },
+              searchParams,
+            ),
+          }}
+        >
+          All
+        </Link>
+
+        <Link
+          type="button"
+          className={classNames({ 'is-active': sex === 'm' })}
```

Comment:
Use enum for 'm' and fix it everywhere
<img width="345" alt="image" src="https://github.com/mate-academy/react_people-table-advanced/assets/64750958/750cbf41-29d9-4d8c-ab68-b29bb83d8f87">